CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
```

Comment:
It's a good suggestion! I think everyone should follow it.

@BobinMathew 
Please, first share your ideas and discuss the changes you wish to make via issue, email, or by contacting the owners of this repository

Reply:
> @BobinMathew
> Please, first share your ideas and discuss the changes you wish to make via issue, email, or by contacting the owners of this repository
ok